lib/utils.pm

```diff
@@ -258,6 +259,23 @@ sub get_netboot_mirror {
     return get_var('MIRROR_' . uc($m_protocol));
 }
 
+sub zypper_call {
+    my ($command, $order) = @_;
+    $order = $order // 1;
+    script_run("zypper -n $command; echo worked-patch-\$?- > /dev/$serialdev", 0);
```

Comment:
better use bmwqemu::hashed_string("ZN$command") instead of worked-patch - so that 2 calls after the other work.